### PR TITLE
Added Virtual Private Cloud Issue#613

### PR DESCRIPTION
**Pull request by akshatcoder-hash**

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@akshatcoder-hash Thank you for your contribution!

Here is a tip:
it seems that you 
1. created a new branch [`akshatcoder-hash:vpc`](https://github.com/akshatcoder-hash/glossary/tree/vpc),
2. merged [`akshatcoder-hash:vpc`](https://github.com/akshatcoder-hash/glossary/tree/vpc) to [`akshatcoder-hash:main`](https://github.com/akshatcoder-hash/glossary/tree/main) using [the PR in your forked repo](https://github.com/akshatcoder-hash/glossary/pull/1),
3. and then opened this PR which merges [`akshatcoder-hash:main`](https://github.com/akshatcoder-hash/glossary/tree/main) to [`cncf:main`](https://github.com/cncf/glossary/tree/main).

Actually you don't need to do like this.
Instead, on `cncf/glossary` repo, open a PR that merges [`akshatcoder-hash:vpc`](https://github.com/akshatcoder-hash/glossary/tree/vpc) to [`cncf:main`](https://github.com/cncf/glossary/tree/main) directly. 😊

File: content/en/virtual-private-cloud.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## What it is
+A virtual private cloud (VPC) is a secure, isolated private cloud hosted within a public cloud. It allows users to do everything thay are able to do on an ordinary cloud, but the private cloud is hosted remotely by a public cloud provider. It gives user the ability to define and control a virtual network that is isolated from all other public cloud customers, creating a private, secure place on the public cloud.
+
+## Problem it addresses
+
+- Security Breaches
+- Performance Issues
+- Overbuying or Underbuying
+- Easier sync, Integration and Deployation
```
Comment (jihoon-seo) [inline content/en/virtual-private-cloud.md:15]: Cases, and 'Deployment' maybe?

```suggestion
- Easier Sync, Integration and Deployment
```

File: content/en/virtual-private-cloud.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- High Carbon Footprint
+
+## How it helps
+
+- In a Virtual Private Cloud, only authenticated user can get access to the information. Moreover, VPCs are monitored for security by Cloud Security Experts regularly.
+- VPC providers update softwares regularly on their end. Thus, a user always runs the latest version of the software and does not need to update it. Thus, the updated software provides better performance.
+- Often, enterprises need to increase their capacity whien they need to scale. VPC providers allow flexibility to the user to modify storage plans accordingly.
```
Comment (jihoon-seo) [inline content/en/virtual-private-cloud.md:22]: Typo: 'whien' -> 'when'

**Review by jihoon-seo — COMMENTED** [review]
PTAL!

**Comment by akshatcoder-hash** [comment]
Hey @jihoon-seo , kindly check the new pull request https://github.com/cncf/glossary/pull/992 . I am closing this one.

**Pull request closed without merging**